docs/source/howto/workflows.rst

```diff
@@ -0,0 +1,29 @@
+.. _how-to:workflows:
+
+*******************************
+How to create and run workflows
+*******************************
+
+
+.. _how-to:workflows:run:
+
+How to run an existing workflow
+===============================
+
+
+.. _how-to:workflows:write-simple:
+
+How to write simple workflows
+=============================
```

Comment:
Should we have an 'how-to' section on how to choose between a WorkChain vs workfunction? (at least implicitly, maybe inside 'how to write simple workflows', that shows both syntaxes and explains - by example - when you pick one or another?)

Or it's really for "topics"? I still think there should be a pointer here, avoiding the assumption that there are only WorkChains (sorry to write this here, it's just not to forget)

Reply:
I agree that there should be a pointer. My idea would be to at the end of the "simple workflow" how-to, quickly explain the limitation of work functions being blocking and non-restartable and that for that reason you might want to use "more complex" workflows which are workchains and then link to that how-to (besides all the other advantages that the work chain gives). I would also add a link to a section in topics that goes into more depth about the difference between work function and work chain, as well as their respective advantages/disadvantages.